src/main/java/org/folio/service/mapping/RuleFactory.java

```diff
@@ -88,6 +89,18 @@
     return rules;
   }
 
+  private void markItemTransformationsWithSameFieldInHoldings(List<Transformations> mappingTransformations) {
+    for (Transformations holdingsTransformation: mappingTransformations) {
+      for (Transformations itemTransformation: mappingTransformations) {
+        if (holdingsTransformation.getRecordType().equals(HOLDINGS) && itemTransformation.getRecordType().equals(ITEM)) {
+          if (holdingsTransformation.getFieldId().equals(itemTransformation.getFieldId())) {
```

Comment:
Wait, Field Id is the path , why will those be same ever? We need to check the tag in the transformation isn't it?

Reply:
fieldId is just an id that looks like 'electronicAccess.linkedText'. It could be in holdings and in items, but the tag could be different